platform-server-kernel/src/main/java/uno/cod/platform/server/core/dto/invitation/InvitationDto.java

```diff
@@ -0,0 +1,36 @@
+package uno.cod.platform.server.core.dto.invitation;
+
+import javax.validation.constraints.NotNull;
+
+public class InvitationDto {
+    @NotNull
+    private String email;
+    @NotNull
+    private Long organizationId;
+    @NotNull
+    private Long challengeId;
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public Long getOrganizationId() {
```

Comment:
since there is already a relation between challenge and organization, the organization id shouldn't be sent